awx/api/serializers.py

```diff
@@ -3609,6 +3639,10 @@ class LaunchConfigurationBaseSerializer(BaseSerializer):
     skip_tags = serializers.CharField(allow_blank=True, allow_null=True, required=False, default=None)
     diff_mode = serializers.BooleanField(required=False, allow_null=True, default=None)
     verbosity = serializers.ChoiceField(allow_null=True, required=False, default=None, choices=VERBOSITY_CHOICES)
+    execution_environment = serializers.PrimaryKeyRelatedField(queryset=ExecutionEnvironment.objects.all(), required=False, allow_null=True, default=None)
```

Comment:
This follows the same pattern as `inventory`, and I don't think it needs to be defined on the serializer, because the DRF serializer does everything we need for this.

Reply:
Removed.